### DISCUSSION TITLE
Can't install if using tlk/imagemagick-x11

### DISCUSSION THREAD

**Discussion by gasparschott** [libvips/libvips#3317] [General]
==> Installing dependencies for vips: imagemagick
Error: imagemagick is already installed from tlk/imagemagick-x11!
Please `brew uninstall imagemagick` first."

I prefer to use this version of imagemagick. Can vips be updated to locate it and install without error?

**Comment by lovell** [comment]
It looks like you're using a custom tap so you might need to ask this question at https://github.com/tlk/homebrew-imagemagick-x11 - I suspect you may have to create your own custom tap also.
